drivers/spi/spi_nxp_lpspi/spi_nxp_lpspi_common.c

```diff
@@ -102,6 +105,11 @@ static inline int lpspi_validate_xfer_args(const struct spi_config *spi_cfg)
 		return -ENOTSUP;
 	}
 
+	if ((spi_cfg->operation & SPI_HOLD_ON_CS) && major_ver < 2) {
+		LOG_ERR("SPI CS hold on feature is not supported on this platform.");
+		return -ENOTSUP;
+	}
```

Comment:
this code should only be in the DMA based driver, it actually is supported on the cpu-based driver.

Reply:
ok, I see.